src/main/java/org/kitteh/irc/client/library/element/User.java

```diff
@@ -100,4 +103,17 @@
      * @return true if away
      */
     boolean isAway();
+
+    /**
+     * Begins a DCC chat with this user.
+     *
+     * <p>When the chat is connected, a {@link DCCConnectedEvent} will be fired. If the connection fails,
+     * a {@link DCCFailedEvent} will be fired.</p>
+     *
+     * @see Client#beginDCCChat(User)
+     */
+    default void beginDCCChat() {
```

Comment:
I'd consider renaming this to requestDccChat over begin, because the latter sort-of implies that the target-user does not have to confirm it. In this case you're requesting a (connection|chat), not beginning it. Might be useful to have some more opinions on this.

(Unless they don't, and I'm just wrong.)